webpack/scenes/AnsibleCollections/AnsibleCollectionsTableSchema.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import {
+  headerFormatter,
+  cellFormatter,
+} from '../../move_to_foreman/components/common/table/formatters';
+
+const TableSchema = [
+  {
+    property: 'name',
+    header: {
+      label: __('Name'),
+      formatters: [headerFormatter],
+    },
+    cell: {
+      formatters: [
+        (value, { rowData }) => (
+          <td>
+            <Link to={urlBuilder('ansible_collections', '', rowData.id)}>{rowData.name}</Link>
+          </td>
+        ),
+      ],
+    },
+  },
+  {
+    property: 'namespace',
+    header: {
+      label: __('NameSpace'),
```

Comment:
`NameSpace` doesn't seem like a user-friendly term, I would change (the user-facing portion of) this to be "Author" or something similar. Let me know if I'm missing some convention though.

I wonder if we should show just list these with the format `author/name`? So the Name column would show `geerlingguy/ntp`. You could even modify the API to return a display name configured like this (and could sort results alphabetically). Just a thought!

Reply:
The` namespace` field name is dicated by the galxy.yml metadata structure: https://docs.ansible.com/ansible/devel/dev_guide/collections_galaxy_meta.html 
There are more fields we plan to index in katello and one of them is `authors`. I think keeping the names consistent with the yml metadata would make it less confusing for now?

Reply:
@Rohoover Can you provide some UX guidance on how to present this information? 

We are adding the ability for users to sync down collections from [ansible galaxy](https://galaxy.ansible.com/home). [Here](https://galaxy.ansible.com/search?tags=system&keywords=&order_by=-relevance) is an example list of collections in ansible galaxy, and [here](https://galaxy.ansible.com/newswangerd/collection_demo) is an example of a collection named "collection_demo"

Right now,  our table looks like this: 
![here](http://www.clipular.com/c/5644278888660992.png?k=mAacMRKWtgjILVC0c-Ar5mRpSQ8)

What do you think is the best way to display these fields?

Reply:
@johnpmitsch Overall this is really consistent with the rest of Satellite. 
A few thoughts"
1) Is "NameSpace" the correct column header? In Ansible the tooltip denotes this content as "Author"
2) Is there a reason we don't show the Quality Score? Is it because of its community origins?
3) The Ansible UI allows you to install different versions, are we mimicking this?
4) I'd recommend an export button at the top.

Reply:
@Rohoover :
> Is "NameSpace" the correct column header? In Ansible the tooltip denotes this content as "Author"

So the exact [description](https://docs.ansible.com/ansible/devel/dev_guide/collections_galaxy_meta.html#collections-galaxy-meta) of the namespace field is: 
```
The namespace of the collection.
This can be a company/brand/organization or product namespace under which all content lives.
May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with underscores or numbers and cannot contain consecutive underscores.
```
I am open to using a more generic term on UI if needed.
****

> Is there a reason we don't show the Quality Score? Is it because of its community origins?

The quality score is not indexed in katello right now. These are newer fields that pulp has now and we plan to index those in a future PR. We do plan to bring it in.

****
> The Ansible UI allows you to install different versions, are we mimicking this?

Yes, a user should be able to sync different versions to his repo if he needs to.

Reply:
> So the exact [description](https://docs.ansible.com/ansible/devel/dev_guide/collections_galaxy_meta.html#collections-galaxy-meta) of the namespace field is:
> 
> ```
> The namespace of the collection.
> This can be a company/brand/organization or product namespace under which all content lives.
> May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with underscores or numbers and cannot contain consecutive underscores.
> ```
> 
> I am open to using a more generic term on UI if needed.

I am in favor of "Author" in the UI since the user is more likely to see it listed as this in Ansible Galaxy than see the namespace metadata field, which I'm not entirely sure the typical user would see. The backend fields could still use `namespace`, I don't see an issue with this.  

But I defer to @rohoover's judgement :)

Reply:
Let's go with "Author". Namespace to me, isn't a real word and parity between the apps will be helpful to users. 

Is the synching of different versions happening in the detailed page upon click of "collection_demo"?
Is each version going to be a separate line item in this table?

Reply:
@Rohoover
> Is the synching of different versions happening in the detailed page upon click of "collection_demo"? Is each version going to be a separate line item in this table?

So the collection_demo in the table would be already synced. If there are multiple versions, they are treated as separate records and would be separate line items in the table.

There's currently some discussions around how we sync in https://pulp.plan.io/issues/5250. We will probably let users specify certain versions when syncing and those versions would get synced.

> Let's go with "Author". Namespace to me, isn't a real word and parity between the apps will be helpful to users.

I will update this to Author.